### ISSUE TITLE
Need custom style for items from score GUI

### ISSUE THREAD

**Issue by jln-** [ossia/score-docs#2]
At the moment, inline code style is used for styling elements from score UI (buttons names, window section etc.). This is not so much sexy and maybe not obvious.

It could be nice and useful that these UI elements are styled in a specific way.

**Comment by aklevy** [comment]
Yep, seems like a good idea ! It should not be difficult to change the css for that but do you have any specific idea as to how it should look like ?

**Comment by jln-** [comment]
I dont really have a defined idea, I must say. I like that css class for keyboard shortcut. Maybe something a bit like this or the Just The Doc button/label css ? Just thinking out loud...

What I am wondering is that we may need to add a specific css class for that so we can still that inline code one for... well inline code. :-)

**Comment by aklevy** [comment]
@jln-  I modified the css, could you check the updated website and tell me if it looks ok to you ?
